### ISSUE TITLE
Temporarily freeze kornia at 0.4.0 to prevent breaking change

### ISSUE THREAD

**Issue by ivanzvonkov** [greentfrapp/lucent#12]
[kornia 0.4.1](https://github.com/kornia/kornia/releases/tag/v0.4.1) released recently and it includes a breaking change to `get_rotation_matrix2d` 

I've opened an Issue to hopefully address this soon: https://github.com/kornia/kornia/issues/742 but in the meantime, I suggest freezing kornia at 0.4.0 so that the `random_rotate` transform continues working as before

**Comment by greentfrapp** [comment]
@ivanzvonkov thanks for this!

**Comment by edgarriba** [comment]
Very nice library guys and super well documented - we are very happy to see Kornia in here :)

Regarding the current issue - why not passing `scale = torch.ones(b, 2)` ?
https://github.com/greentfrapp/lucent/blob/master/lucent/optvis/transform.py#L71

An alternative is to use [`kornia.rotate`](https://kornia.readthedocs.io/en/latest/geometry.transform.html?highlight=rotate#kornia.geometry.transform.rotate) which doesn't take into account the scale as input.

Besides, I detected few other functionalities in your code that could be used directly with kornia. Let me list them:
- `kornia.deg2rad`  https://github.com/greentfrapp/lucent/blob/master/lucent/optvis/transform.py#L69
- `kornia.tensor_to_image` https://github.com/greentfrapp/lucent/blob/master/lucent/optvis/render.py#L132
- `kornia.normalize` https://github.com/greentfrapp/lucent/blob/master/lucent/optvis/transform.py#L106

Last, what's the intuition or if you have any reference for [`_linear_decorrelate_color`](https://github.com/greentfrapp/lucent/blob/master/lucent/optvis/param/color.py#L33) ?
I believe with a proper API design would be great to have it in kornia.
